### ISSUE TITLE
Tamaño de archivo L4_1

### ISSUE THREAD

**Issue by tgongorap** [IIC2115/Syllabus#22]
Hola! Acabo de terminar de realizar la parte 1 del laboratorio 4 y no me deja subir el archivo a GitHub por el tamaño de este (pesa 500 mb aproximadamente.) ¿Que se hace en este caso?

**Comment by mariapazmunoz** [comment]
Holaa! también me sucede eso y no puedo subir el laboratorio ;( Me sale que git acepta como máximo un archivo de 100 mb

**Comment by TrinidadVera** [comment]
A mi tampoco me deja subirlo, github solo me permite subir archivos que pesan menos de 25 MB y mi laboratorio pesa más.

**Comment by mariapazmunoz** [comment]
Holaa! Eliminando el gráfico pude subirlo pero no se muestra con previsualización en git. Me sale esto 
<img width="1105" alt="image" src="https://github.com/user-attachments/assets/dfea5f47-cad7-427b-a8dd-5a7f332d8639"> Se puede descargar y abrir en aplicaciones que soporten .ipynb (como VSCode), es la única forma que lo pude subir ;(

**Comment by fmandiolafellay** [comment]
Los mapas .html comprimidos al subirlos igual falla, a pesar de que el archivo pesa 50 megas git reconoce el peso de los mapas igual, pasando los 100mb

**Comment by fmandiolafellay** [comment]
No, por alguna razón github dice que el codigo pesa 150mb y en mis archivos pesa 1mb (con las celdas de los mapas eliminados) y lo rechaza. No soy al unico que le pasa.

**Comment by IsidoraBerrios** [comment]
Hola @plourido también me pasó, aún no he podido subirlo
